x-pack/plugins/monitoring/public/angular/providers/private.js

```diff
@@ -81,9 +81,9 @@
  *
  * @param {[type]} prov [description]
  */
-import _ from 'lodash';
+import { partial, uniqueId, isObject } from 'lodash';
```

Comment:
FYI, this has no impact on the size of the bundles since we always load and share a single, complete, lodash instance.